napari/_qt/qt_viewer.py

```diff
@@ -571,6 +571,22 @@ def screenshot(self, path=None):
             imsave(path, img)  # scikit-image imsave method
         return img
 
+    def clipboard(self):
+        """Take a screenshot of the currently displayed screen and copy the
+        image to the clipboard.
+        """
+        from .utils import add_flash_animation
+
+        img = self.canvas.native.grabFramebuffer()
```

Comment:
can we maybe put this line inside a private `_screenshot` method that we then use inside our public `screenshot(self, path=None)` and here too (i.e. so as not to duplicate `img = self.canvas.native.grabFramebuffer()`). I'd like to keep instances of going into `.native` as few as possible in the code.

Also I'm wondering if the flash shouldn't be inside that method too, could be a parameter

```
def _screenshot(self, flash=True):
...
    returns img
```

Reply:
This is great (as usual) @lukasz-migas!  @sofroniewn's comment here was my only comment/thought as well: namely, that `clipboard()` and `screenshot()` methods should be using the same function behind the scenes to actually get the image.  Otherwise, great stuff!

Reply:
Thanks for the feedback - I've refactored the `clipboard()` and `screenshot()` functions to use the same private method behind the scenes. I've also added `flash=True` to the `screenshot()` method to highlight when screenshot has taken place.